pybotvac/account.py

```diff
@@ -29,52 +29,52 @@
         Required("serial"): str,
         "prefix": Any(str, None),
         Required("name"): str,
-        "model": str,
+        "model": Any(str, None),
         Required("secret_key"): str,
         "purchased_at": Any(str, None),
         "linked_at": Any(str, None),
         Required("traits"): list,
         # Everything below this line is not documented, but still present
-        "firmware": str,
-        "timezone": str,
+        "firmware": Any(str, None),
+        "timezone": Any(str, None),
         Required("nucleo_url"): Url,
-        "mac_address": str,
-        "created_at": str,
+        "mac_address": Any(str, None),
+        "created_at": Any(str, None),
     },
     extra=ALLOW_EXTRA,
 )
 MAP_SCHEMA = Schema(
     {
-        "version": int,
+        "version": Any(int, None),
         Required("id"): str,
         Required("url"): Url,
-        "url_valid_for_seconds": int,
-        Optional("run_id"): str,  # documented, but  not present
+        "url_valid_for_seconds": Any(int, None),
+        Optional("run_id"): str,  # documented, but not present
         "status": Any("complete", "incomplete", "canceled"),
         "launched_from": Any("robot", "schedule", "app"),
```

Comment:
Should we edit the fields `status` and `launched_from` to be less restrictive as well?

```suggestion
        "status": Any(str, None),
        "launched_from": Any(str, None),
```

Reply:
I'd say that unless we depend on the value of the fields in downstream code, we should be as permissive as possible. As far as I can tell that is the case for both `status` and `launched_from`.

Reply:
I agree. Nice!